python/src/_builder.py

```diff
@@ -174,8 +174,10 @@ def build_memory_index(
     num_pq_bytes: int = defaults.NUM_PQ_BYTES,
     use_opq: bool = defaults.USE_OPQ,
     vector_dtype: Optional[VectorDType] = None,
-    filter_complexity: int = defaults.FILTER_COMPLEXITY,
     tags: Union[str, VectorIdentifierBatch] = "",
+    filter_labels: Optional[list[list[Hashable]]] = None,
```

Comment:
I'm not even confident this is the right signature, since it's just being serialized to a file anyway. Should I change this to `Optional[list[list[str]]]`?

Reply:
It should probably just be str, why do you think it should be Hashable?